docs/components/compact.md

```diff
@@ -55,18 +55,27 @@ In fact, downsampling doesn't save you any space but instead it adds 2 more bloc
 
 ## Groups
 
-The compactor groups blocks using the external_labels added by the Prometheus who produced the block.
-The labels must be both _unique_ and _persistent_ across different Prometheus instances.
+The compactor groups blocks using the external*labels added by the Prometheus who produced the block.
+The labels must be both \_unique* and _persistent_ across different Prometheus instances.
 
 By _unique_, we mean that the set of labels in a Prometheus instance must be different from all other sets of labels of
 your Prometheus instances, so that the compactor will be able to group blocks by Prometheus instance.
 
 By _persistent_, we mean that one Prometheus instance must keep the same labels if it restarts, so that the compactor will keep
 compacting blocks from an instance even when a Prometheus instance goes down for some time.
 
+## Block Deletion
+
+Depending on the Object Storage provider like S3, GCS, cepth hdfs etc; the storage can be strongly consistent or eventually consistent. This poses a challenge to co-ordinate readers and writers of the block.
+Since there are no consistency guarantees provided by Object Storage provider, we have to make sure that we have a consistent lock-free way of dealing with Object Storage irrespective of the choice of object storage.
```

Comment:
```suggestion
Since there are no consistency guarantees provided by some Object Storage provider, we have to make sure that we have a consistent lock-free way of dealing with Object Storage irrespective of the choice of object storage.
```

Reply:
sure done